crates/hyperswitch_connectors/src/connectors/novalnet/transformers.rs

```diff
@@ -425,13 +472,13 @@ pub enum NovalnetResponsePaymentData {
 
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct NovalnetResponseCard {
-    pub card_brand: Secret<String>,
+    pub card_brand: Option<Secret<String>>,
     pub card_expiry_month: Secret<u8>,
     pub card_expiry_year: Secret<u16>,
     pub card_holder: Secret<String>,
     pub card_number: Secret<String>,
-    pub cc_3d: Secret<u8>,
-    pub last_four: Secret<String>,
+    pub cc_3d: Option<Secret<u8>>,
+    pub last_four: Option<Secret<String>>,
     pub token: Option<Secret<String>>,
```

Comment:
Make token as `Option<String>` because `mandate_reference.connector_mandate_id` is also `Option<String>`